### ISSUE TITLE
Visibility binding property doesn't work

### ISSUE THREAD

**Issue by cxs6056** [michaelpapworth/tinymce-knockout-binding#5]
The visibility property only works on the original element, not the element created by TinyMCE . 

For example

``` html
<div data-bind="foreach: cards">
    <textarea class="tinymce" data-bind="wysiwyg: text, visible: isCurrentCard"></textarea>
</div>
```

When isCurrentCard is set to true, the textarea element will appear but the `<div>` created by TinyMCE will not. The `<textarea>` should remain hidden and the new TinyMCE control should be displayed instead.

Thanks for the great work.

Best,

Chuck

**Comment by michaelpapworth** [comment]
@cxs6056 Thanks for raising this. I'll ensure that it features in my next release.  In the meantime, may I suggest the following workaround?

``` html
<div data-bind="foreach: cards">
    <!-- ko if: isCurrentCard -->
    <textarea class="tinymce" data-bind="wysiwyg: text"></textarea>
    <!-- /ko -->
</div>
```

**Comment by cxs6056** [comment]
Thanks Michael!

I'm experimenting with using CKEdit now.

**Comment by michaelpapworth** [comment]
The work around I provided in my previous comment is in fact the correct way to apply the if binding in this scenario according to [the documentation](http://knockoutjs.com/documentation/if-binding.html).  The fault lies with tinymce-knockout-binding not controlling descendent bindings.  This is now fixed as part of v1.1.0. and if you try to apply the `if` or `ifnot` binding in conjunction with the `wysiwyg` binding an error is thrown.
